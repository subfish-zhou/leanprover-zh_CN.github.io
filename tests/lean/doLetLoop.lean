set_option showPartialSyntaxErrors true
def f : IO Unit := do
  if let
