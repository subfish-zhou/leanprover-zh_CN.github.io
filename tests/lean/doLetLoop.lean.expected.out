doLetLoop.lean:4:0: error: unexpected end of input
